docs/sphinx/development.rst

```diff
@@ -18,8 +18,42 @@ Additional Poetry documentation:
 Requirement: Python version 3.8 or greater
 ------------------------------------------
 
-Installation
-------------
+Installation Using a Python Environment 
+---------------------------------------
+To install Poetry using a python environment, you must first set up the environment using the following
+commands (Please note that the name of the environment is not limited to beedev-en). All of the
+commands should be run in your home directory. Make sure your default shell is set to bash by running
+``echo $SHELL``. If it is not in bash, run ``chsh -s /bin/bash`` and enter your personal device password.
+
+If you prefer your default shell to be in zsh and the following directions give you errors, use
+``pip3`` instead of ``pip``.     
+
+.. code-block::
+
+    mkdir beedev-env
+    python3 -m venv beedev-env
+    source beedev-env/bin/activate 
+    pip install poetry
+
+To activate the python environment and build beeflow:
+
+.. code-block::
+
+    source beedev-env/bin/activate
+    cd <path to BEE repo>
+    poetry install
+    beeflow --version
+
+If you want to exit the python environment, enter the following command:
+
+.. code-block::
+    
+    deactivate
+
+You can make sure Poetry is installed by using the following command: ``poetry --version``.
+
+Additional Installation
+-----------------------
 It is possible to install Poetry using Pip, but it is recommended to instead
```

Comment:
Since you added directions to install poetry with pip you should delete line 57. Also, I'm not sure I would give directions for zsh. Instead maybe just say these are the instruction if your shell is bash and may differ for others.